src/stores/store-urls-custom-admin.md

```diff
@@ -51,27 +51,62 @@ As a precaution, do not try to change the Admin URL by yourself unless you know
 
 1. After the changes are saved, **Sign Out** of the Admin. Then, log back in using the new Admin URL and path.
 
-## Method 2: Change from the Server Command Line
+## Method 2: Change the Admin Path from the server command line
 
-1. Open the `app/etc/env.php` file in a text editor, and change the name of the `[admin]` path. Make sure to use only lowercase characters. Then, save the file.
+1. Open the `app/etc/env.php` file in a text editor, and change the value of the `frontName` parameter of the `backend` section. Then, save the file.
 
-    On the server, the _admin path_ is located in the `app/etc/env.php` file. Look for the `<adminhtml>` argument in the `<admin>` section:
+   Make sure to use only lowercase characters.
 
-   - **Default Admin Path**
+   {:.bs-callout-info}
+   This method allows you to change the Admin Path, but not the Admin URL.
 
-        ```php?start_inline=1
-        # <frontName><![CDATA[admin]]></frontName>
-        ```
-        {: .no-copy}
+   - **Default Admin Path**
+      ```php?start_inline=1
+      'backend' => [
+       'frontName' => 'admin'
+      ],
+      ```
+      {: .no-copy}
 
    - **New Admin Path**
+      ```php?start_inline=1
+      'backend' => [
+          'frontName' => 'backend'
+      ],
+      ```
+      {: .no-copy}
+
+1. Use one of the following methods to clear the Magento cache:
+
+   - On the _Admin_ sidebar, go to **System** > _Tools_ > **Cache Management**. Then, click **Flush Magento Cache**.
+   - On the server, execute the following:
+      ```terminal
+      php bin/magento cache:flush
+      ```
+
+   {:.bs-callout-info}
+   Note that the changes made using the Method 1 will have the priority over the changes made in the `app/etc/env.php` file.
```

Comment:
```suggestion
   Note that the changes made using Method 1 will have priority over the changes made in the `app/etc/env.php` file.
```

Reply:
good catch... will have to go back and fix that in master